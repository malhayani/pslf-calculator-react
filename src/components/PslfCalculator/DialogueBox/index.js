import React from 'react';

const DialogueBox = (props) => {
    return (
        <div>
            <label className="dialogue">{ props.msg }</label>
        </div>
    )
}

export default DialogueBox;