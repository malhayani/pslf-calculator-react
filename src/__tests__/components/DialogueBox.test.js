import React from 'react';
import DialogueBox from '../../components/DialogueBox';
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
Enzyme.configure({ adapter: new Adapter() })

describe('DialogueBox', () => {
    it('renders correctly', () => {
        const wrapper = Enzyme.shallow(<DialogueBox />);
        expect(wrapper).toMatchSnapshot();
    });

    it('Expect dialogue box displays text', () => {
        const wrapper = Enzyme.shallow(<DialogueBox msg="Test"/>);
        expect(wrapper.find('.dialogue').html().includes('Test')).toEqual(true);
    });
});