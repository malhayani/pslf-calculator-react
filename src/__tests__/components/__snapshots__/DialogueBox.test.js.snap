// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogueBox renders correctly 1`] = `ShallowWrapper {}`;
